src/modules/AddQR/Module.js

```diff
@@ -0,0 +1,50 @@
+module.exports = function AddQR(options, UI) {
+
+    options.size = options.size || 200;
+    options.qrCodeString = options.qrCodeString || "https://github.com/publiclab/image-sequencer";
```

Comment:
Should this default be in the `info.json` file now?

Reply:
Sure done that!